src/packagedcode/go_mod.py

```diff
@@ -250,15 +250,19 @@ def parse_gomod(location):
 # Regex expressions to parse go.sum file dependency
 # dep example: github.com/BurntSushi/toml v0.3.1 h1:WXkYY....
 get_dependency = re.compile(
-    r'(?P<namespace>[^\s]*)'
-    r'\/'
     r'(?P<name>[^\s]*)'
```

Comment:
what if you named this `ns_name` or `namespace_name` rather than `name` since this is what it is now?